### PR TITLE
Docs revamp edits

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
@Blackbaud-LindseyRix, can you please take a quick look at my changes to your stuff to make sure everything looks OK? Hopefully this is faster than going through feedback in comments and deciding which changes you want to make. Thanks!

### CODE REVIEW

File: src/app/support/troubleshoot/index.html [hunk 1]
```diff
@@ -57,53 +57,52 @@ <h3>
...
+      Did you misspell a component? For example, if you reference <stache-code>stace-code</stache-code> instead of <stache-code>stache-code</stache-code>, your SPA will try to find that misspelled component and fail.
     </li>
   </ul>
 
   <p>
-    If you've made a recent change, but everything looks good, you can also stop serving and serve again to see if that fixes
-    the issue. If it didn't, keep looking, as there's an error in there somewhere. Still can't find it?
-    <a routerLink="/support">Contact us</a> and we'd be glad to help hunt it down.
+    If you've made a recent change and everything looks good, you can also stop serving and serve again to see if that fixes
+    the issue. Ifnot, keep looking because there's an error in there somewhere. Still can't find it?
```
Comment (Blackbaud-LindseyRix) [inline src/app/support/troubleshoot/index.html]: needs a space

File: src/app/learn/basics/definition-list-demo.component.ts [hunk 2]
```diff
@@ -8,47 +8,47 @@ export class SkyDefinitionListDemoComponent {
...
     },
     {
       label: 'data',
-      value: 'This is a place to include JSON data that you can use in your application.'
+      value: 'This folder can contain JSON data to use in your application.'
     },
     {
       label: 'global.json',
-      value: 'This is where you set global data variables, like your application title.'
+      value: 'This file is where you set global data variables such as the application title.'
```
Comment (Blackbaud-LindseyRix) [inline src/app/learn/basics/definition-list-demo.component.ts]: comma after variables?

File: src/app/support/troubleshoot/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/support/troubleshoot/index.html]: Fixed

File: src/app/learn/basics/definition-list-demo.component.ts (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/learn/basics/definition-list-demo.component.ts]: Added a comma.

**Pull request merged**